ui/src/app/app-routing.module.ts

```diff
@@ -380,22 +378,25 @@ const routes: Routes = [
           },
           {
             path: 'workspaces',
-            component: AdminWorkspaceSearchComponent,
-            data: { title: 'Workspace Admin'},
+            component: AppRouting,
```

Comment:
optional nit: existing issue but these migrated routes probably should go under this section instead: "// legacy / duplicated routes go HERE"

Reply:
oops I forgot I did that before! I will do that and add it to the epic ticket